src/pages/Dashboard/Dashboard.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Container from '@mui/material/Container';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import { useDispatch } from 'react-redux';
+import { Logout } from '../../features/userSlice';
+
+function Dashboard() {
+  const dispatch = useDispatch();
+
+  const handleLogout = (e) => {
+    e.preventDefault();
+
+    dispatch(Logout());
+  };
+
+  return (
+    <React.Fragment>
+      <Container fixed>
+        <Box sx={{ height: '100vh', display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+          <Button
+            variant="contained"
+            color="error"
```

Comment:
`Error` color for logout? Is it intentionally or what? `Error` suggest rather a mistake then `intentional action`